tests/unit/lms/views/api/grading_test.py

```diff
@@ -203,3 +206,8 @@ def pyramid_request(self, pyramid_request):
             "score": 0.5,
         }
         return pyramid_request
+
+
+@pytest.fixture
+def LTIEvent(patch):
+    return patch("lms.views.api.grading.LTIEvent")
```

Comment:
I think this fixture should be kept local to the tests that use it. It's only used in `TestRecordCanvasSpeedgraderSubmission`.

The reason for this is it makes it easier to refactor later, as you can pick up the whole class and be more sure you have everything you need.

The other direction some of our tests go is a big soup of fixtures at the bottom, which more often than not, are local to a specific set of tests which have to be teased apart.